index.html

```diff
@@ -45,5 +45,117 @@ <h2>LET'S CONNECT</h2>
       </ul>
       
     </section>
+    <section class="cardworks-one">
+  <img src="./images/Snapshoot Portfolio.png" alt="" class="snapshoot">
+  <div class="left-blockone"></div>
+  <h1>Multi-post stories</h1>
```

Comment:
- Kindly make sure you have h1 only on once on your page to respect the semantic HTML rules :recycle: